internals/compiler.rst

```diff
@@ -524,140 +524,112 @@ statement in ``_PyEval_EvalFrameDefault()``.
-        code.
+   * :cpy-file:`Parser/pegen.h`: Header file for the corresponding
+     :cpy-file:`Parser/pegen.c`. Also contains definitions of the ``Parser``
+     and ``Token`` structs.
 
-    pegen.h
-        Header file for the corresponding :cpy-file:`Parser/pegen.c`. Also contains
-        definitions of the ``Parser`` and ``Token`` structs.
+* :cpy-file:`Python/`
+   * :cpy-file:`Python/Python-ast.c`: Creates C structs corresponding to
+     the ASDL types.  Also contains code for marshalling AST nodes (core
+     ASDL types have marshalling code in :cpy-file:`Python/asdl.c`).
+     "File automatically generated by :cpy-file:`Parser/asdl_c.py`".
+     This file must be committed separately after every grammar change
+     is committed since the ``__version__`` value is set to the latest
+     grammar change revision number.
 
-+ Python/
+   * :cpy-file:`Python/asdl.c`: Contains code to handle the ASDL sequence type.
+     Also has code to handle marshalling the core ASDL types, such as number
+     and identifier.  Used by :cpy-file:`Python/Python-ast.c` for marshalling
+     AST nodes.
 
-    Python-ast.c
-        Creates C structs corresponding to the ASDL types.  Also
-        contains code for marshalling AST nodes (core ASDL types have
-        marshalling code in :cpy-file:`Python/asdl.c`).  "File automatically
-        generated by :cpy-file:`Parser/asdl_c.py`".  This file must be
-        committed separately after every grammar change is committed since
-        the ``__version__`` value is set to the latest grammar change
-        revision number.
+   * :cpy-file:`Python/ast.c`: Used for validating the AST.
 
-    asdl.c
-        Contains code to handle the ASDL sequence type.  Also has code
-        to handle marshalling the core ASDL types, such as number and
-        identifier.  Used by :cpy-file:`Python/Python-ast.c` for marshalling
-        AST nodes.
+   * :cpy-file:`Python/ast_opt.c`: Optimizes the AST.
 
-    ast.c
-        Used for validating the AST.
+   * :cpy-file:`Python/ast_unparse.c`: Converts the AST expression node
+     back into a string (for string annotations).
 
-    ast_opt.c
-        Optimizes the AST.
+   * :cpy-file:`Python/ceval.c`: Executes byte code (aka, eval loop).
 
-    ast_unparse.c
-        Converts the AST expression node back into a string
-        (for string annotations).
+   * :cpy-file:`Python/compile.c`: Emits bytecode based on the AST.
 
-    ceval.c
-        Executes byte code (aka, eval loop).
+   * :cpy-file:`Python/symtable.c`: Generates a symbol table from AST.
 
-    compile.c
-        Emits bytecode based on the AST.
+   * :cpy-file:`Python/peephole.c`: Optimizes the bytecode.
 
-    symtable.c
-        Generates a symbol table from AST.
+   * :cpy-file:`Python/pyarena.c`: Implementation of the arena memory manager.
 
-    peephole.c
-        Optimizes the bytecode.
+   * :cpy-file:`Python/wordcode_helpers.h`: Helpers for generating bytecode.
```

Comment:
https://github.com/python/cpython/blob/main/Python/wordcode_helpers.h

[This file](https://github.com/python/cpython/blob/18b5dd68c6b616257ae243c0b6bb965ffc885a23/Python%2Fwordcode_helpers.h) was deleted ([on Feb 25](https://github.com/python/cpython/commit/0f41aac109c45aa468c432f798947c54d4178b3d)) - [Commit history](https://github.com/python/cpython/commits/main/Python/wordcode_helpers.h).